doc/compiler/runtime/J9JITExceptionTable.md

```diff
@@ -0,0 +1,437 @@
+    |---------------------------------------------------|
+(5) | Internal Pointer Map in J9 Format                 |
+    |---------------------------------------------------|
+(6) | OSR Info                                          | // If OSR
+    |---------------------------------------------------|
+(7) | GPU Info                                          | // If GPU
+    |---------------------------------------------------|
+(8) | Bytecode PC to Instruction Address Map            | // If Hardware Profiling
+    +---------------------------------------------------+
+```
+
+## (1) Exception Info
+The Exception Info section starts immediately after the
+`J9JITExceptionTable` struct, and so accessing it involves adding
+`sizeof(J9JITExceptionTable)` bytes to a pointer pointing to the start
+of the metadata. This section consists of the following set of data
+for exception in the method body
+```
+    +---------------------------------------------------+
+    | Instruction Start PC Offset                       |
+    | Size: 2 or 4 bytes                                |
+    |---------------------------------------------------|
+    | Instruction End PC Offset                         |
+    | Size: 2 or 4 bytes                                |
+    |---------------------------------------------------|
+    | Instruction Handler PC Offset                     |
+    | Size: 2 or 4 bytes                                |
+    |---------------------------------------------------|
+    | Catch Type                                        |
+    | Size: 2 or 4 bytes                                |
+    |---------------------------------------------------|
+    | BC Caller Index                                   | // If wide offsets, and AOT or Remote Compilation
+    | Size: pointer                                     |
+    |---------------------------------------------------|
+    | J9Method                                          | // If wide offsets, and local JIT Compilation
+    | Size: pointer                                     |
+    |---------------------------------------------------|
+    | BC Index                                          | // If FSD
+    | Size: 4 bytes                                     |
+    +---------------------------------------------------+
+```
+
+## (2) Inline Info
+The Inline Info section can be accessed via the `inlinedCalls` field of
+the `J9JITExceptionTable` struct. This section consists of the following
+set of data for each inlined site
+```
+    +---------------------------------------------------+
+    | struct TR_InlinedCallSite                         |
+    | Size: sizeof(TR_InlinedCallSite)                  |
+    |---------------------------------------------------|
+    | Monitor Autos Map                                 |
+    | Size: Bytes needed to represent                   |
+    |       cg->getStackAtlas()                         |
+    |         ->getNumberOfSlotsMapped()                |
```

Comment:
Should these be on the same line for readability?  My brain was looking for a missing arithmetic operator because of the way the text is stylized here, but obviously one doesn't apply here.

Same for all the tables below.